layers/meta-balena-genericx86/recipes-kernel/linux/kernel-image-initramfs.bbappend

```diff
@@ -0,0 +1,9 @@
+do_install_append_genericx86-64-ext () {
+    if [ "${SIGN}" != "true" ]; then
```

Comment:
In most cases this can be implicitely deduced. For example, on other recipes that made use of it the logic could be extracted to a class.
Let's try to find a way not to have need for this `SIGN` variable.

Reply:
This is supposed to be a global toggle - even if we have all the metadata defined, we want to be able to build an unsigned release.